packages/esm-patient-test-results-app/src/panel-view/panel.component.tsx

```diff
@@ -73,24 +73,16 @@ const LabSetPanel: React.FC<LabSetPanelProps> = ({ panel, observations, activePa
             id: test.id,
             testName: test.name,
             value: {
-              content: (
-                // <span className={getClass(test.obs?.[0]?.interpretation)}>
-                <span>{test.value}</span>
-                // </span>
-              ),
+              content: <span>{`${test.value} ${test.meta?.units}`}</span>,
             },
             interpretation: test.interpretation,
-            range: test.meta?.range ?? '--',
+            range: `${test.meta?.range} ${test.meta?.units}` ?? '--',
```

Comment:
Isn't it weird to condition on strings, even if it is undefined it will be 'undefined' which looks like a value, you should probably condition on the prop and then display a value else you display the `emDash`

Reply:
Thanks for the catch!
I'll fix this asap.